givitsite/friendreq/models.py

```diff
@@ -3,6 +3,9 @@
 # Create your models here.
 
 class ItemRequest(models.Model):
+    friend_id = models.IntegerField()
     item = models.CharField(max_length = 40)
     city = models.CharField(max_length = 40)
-    special_req= models.TextField()
\ No newline at end of file
+    special_req= models.TextField()
+    isOpen = models.BooleanField(default = True)
+    
```

Comment:
what editor are you using. I don't know why but it doesn't add `newline` at the end of file.

Reply:
I use visual studio.

Reply:
I believe you can configure it to add newline at end of file the same way you do it in vs code.
Go to File -> Preferences -> Settings in the "Search settings" box enter "insert final newline" and check it. Then save again this file and see in the diff (use `git diff`) to make sure the `newline` character was added.

Reply:
can you please explain what do you mean by a new line.
because it seems that there is a new line in givitsite/friendreq/models.py.
(line 11)

Reply:
Go to the "Files changed" tab and you can see this warning:
![www](https://user-images.githubusercontent.com/2453279/91471435-c4a75980-e89e-11ea-8db7-dfcde22ca878.png)

Here is a thread on StackOverflow what is the deal with "newline" in git: https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
I have done what you have said, but it doesn't find any changed in my files, therefore, I cannot recommit.
what I type "git diff" it looks like nothing happened

![image](https://user-images.githubusercontent.com/61931613/91472456-20261700-e8a0-11ea-97e6-e0fdb6238841.png)

![image](https://user-images.githubusercontent.com/61931613/91472526-4055d600-e8a0-11ea-9e8c-f760f787ba08.png)

Reply:
Let's troubleshoot this in Slack. :smiley: